package.json

```diff
@@ -131,6 +131,7 @@
     "eslint-plugin-react": "^7.1.0",
     "jest": "^21.2.1",
     "js-yaml": "^3.9.0",
+    "jsdoc": "^3.5.5",
```

Comment:
Since this is already installed globally, this can be removed.

Reply:
Actually, I take that back.  If our tests rely on jsdoc being there, we should probably leave that in `devDependencies` so users won't need the global install to be able to get tests to pass.  So ignore the previous comment.